telemetry/definitions/commonDefinitions.json

```diff
@@ -3294,6 +3438,419 @@
+            "description": "An interactivity measured when users cancel a patch view.",
+            "metadata": [
+                {
+                    "type": "codeTransformPatchViewerCancelSrcComponents",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformRuntimeError",
+                    "required": false
+                },
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformJobId",
+                    "required": false
+                }
+            ]
+        },
+        {
+            "name": "codeTransform_vcsViewerSubmitted",
+            "description": "An interactivity measured when users submit their patch view results.",
+            "metadata": [
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformJobId",
+                    "required": false
+                }
+            ]
+        },
+        {
+            "name": "codeTransform_logApiLatency",
+            "passive": true,
+            "description": "A general logging measure for all client side API's latency.",
+            "metadata": [
+                {
+                    "type": "codeTransformApiNames",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformRunTimeLatency",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformRequestId",
+                    "required": false
+                },
+                {
+                    "type": "codeTransformJobId",
+                    "required": false
+                },
+                {
+                    "type": "codeTransformUploadId",
+                    "required": false
+                },
+                {
+                    "type": "codeTransformTotalByteSize",
+                    "required": false
+                }
+            ]
+        },
+        {
+            "name": "codeTransform_logGeneralError",
+            "passive": true,
+            "description": "A general logging measure for all client side API errors.",
+            "metadata": [
+                {
+                    "type": "codeTransformApiErrorMessage",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                },
+                {
+                    "type": "codeTransformJobId",
+                    "required": false
+                }
+            ]
+        },
+        {
```

Comment:
This is passive because a lot of our API's are not directly triggered by a click, but may happen post click and analysis.